lib/iris/tests/test_trajectory.py

```diff
@@ -189,13 +189,10 @@ def test_hybrid_height(self):
         # Put a biggus array on the cube so we can test deferred loading.
         cube.lazy_data(biggus.NumpyArrayAdapter(cube.data))
 
-        traj = (('grid_latitude', [20.5, 21.5, 22.5, 23.5]),
+        traj = (('grid_latitude', [20.499, 21.501, 22.501, 23.501]),
                 ('grid_longitude', [31, 32, 33, 34]))
-        xsec = iris.analysis.trajectory.interpolate(cube, traj, method='nearest')
 
```

Comment:
I have altered the numbers here slightly so that the test case is not so unstable.  In the reference, the nearest neighbour scheme rounded the grid latitudes to [20, 22, 23, 24].  This should ensure that the standard nearest neighbour scheme yields the same result.